base16-shell.plugin.bash

```diff
@@ -2,5 +2,5 @@ script_path=${BASH_SOURCE[0]}
 script_path=$(readlink -f $script_path)
 BASE16_SHELL_PATH=${script_path%/*}
 
-[ -n "$PS1" ] && [ -s $BASE16_SHELL_PATH/profile_helper.sh ] && source \
+[ -n "$PS1" ] && [ -s $BASE16_SHELL_PATH/profile_helper.sh ] && \
```

Comment:
`source` unnecessary, since `profile_helper.sh` is already executable
also, with `exit` replacing `return` this file can't be sourced, or it exits the current shell

Reply:
I believe the error is coming from bash, though:
```
❱ cat foo.sh                                                 2
#! /bin/bash

return 1
❱ ./foo.sh
./foo.sh: line 3: return: can only `return' from a function or sourced script
❱ bash foo.sh                                                2
foo.sh: line 3: return: can only `return' from a function or sourced script
❱ zsh foo.sh                                                 2
❱ echo $?                                                    1
1
```

Reply:
though, `zsh` is only reflecting the `return` code of `1` - it isn't a syntax error like in `bash`

Reply:
Ok you can ignore the fish changes for this PR, but I still want to merge it with the `./hooks/*.sh` and `profile_helper.sh:68` updated to match these changes.

Reply:
no worries - I'll revert the `fish` changes :)